### ISSUE TITLE
Machine UIs

### ISSUE THREAD

**Issue by fzzhd123** [MineMaarten/PneumaticCraft#573]
For several of the machines, the UIs are wonky. You have to go into the mine craft setting and make the interface smaller to be able to see/use the interface. For example, the bottom row of plastics can not be selected unless I change the UI size.

**Comment by MineMaarten** [comment]
You're talking about the Plastic Mixer I guess?

**Comment by fzzhd123** [comment]
Yea sorry i forgot to add that. I know the liwuud compressor suffers the same problem. You cant read some of the tabs.

**Comment by fzzhd123** [comment]
Liquid lol

**Comment by MineMaarten** [comment]
Yeah, I've noticed the problem. It's quite hard to solve though, lol :P. But I'll see.

**Comment by fzzhd123** [comment]
Ah okay. I was just making sure you knew about it
